driver-core/src/test/unit/com/mongodb/internal/connection/LoggingCommandEventSenderSpecification.groovy

```diff
@@ -109,25 +109,29 @@ class LoggingCommandEventSenderSpecification extends Specification {
 
         then:
         1 * logger.debug {
-            it == "Sending command \'{\"ping\": 1, \"\$db\": \"test\"}\' with request id ${message.getId()} to database test " +
-                    "on connection [connectionId{localValue:${connectionDescription.connectionId.localValue}}] " +
-                    'to server 127.0.0.1:27017'
+            it == "Command \"ping\" started on database test using a connection with driver-generated ID " +
```

Comment:
This is what pushed me over the edge on disabling the unnecessary gstring codenarc rule

Reply:
Huzzah! I hated that rule back when all tests were groovy tests - it always caught me out.